src/commands/init.ts

```diff
@@ -85,6 +85,11 @@ choices.push({
   name: "Data Connect: Set up a Firebase Data Connect service.",
   checked: false,
 });
+choices.push({
+  value: "dataconnect:sdk",
+  name: "Data Connect: Set up a generated SDK for your Firebase Data Connect service.",
```

Comment:
Data Connect SDK: Set up a generated SDK for your Firebase Data Connect service.

<img width="687" alt="Screenshot 2024-06-25 at 5 28 38 PM" src="https://github.com/firebase/firebase-tools/assets/9068391/f93ca70e-6ff5-427b-8095-018f4aa8133b">

Reply:
Followed the pattern used by hosting here, which has 2 init options:
<img width="680" alt="Screenshot 2024-06-25 at 6 05 22 PM" src="https://github.com/firebase/firebase-tools/assets/4635763/60724c7a-9c50-4247-996a-fb3bbcc79489">